src/scripts/main.js

```diff
@@ -1 +1,87 @@
-'use strict';
+const body = document.querySelector('body');
+
+function firstPromise() {
+  return new Promise((resolve, reject) => {
+    let a = 0;
+
+    const timeOut = setTimeout(() => {
+      const errorMessage =
+        '<div data-qa="notification" class="error">' +
+        'First promise was rejected</div>';
+
+      body.insertAdjacentHTML('afterbegin', errorMessage);
```

Comment:
it's better to create div element and set it's attribute (data-qa), class and text